src/libs/ReportUtils.ts

```diff
@@ -5124,7 +5116,12 @@ function getIOUReportActionDisplayMessage(reportAction: OnyxEntry<ReportAction>)
         return Localize.translateLocal(translationKey, {amount: formattedAmount, payer: ''});
     }
 
-    const transaction = getTransaction(originalMessage.IOUTransactionID ?? '');
+    // This log to server is temporary and needed to determine if there is a case we need the transaction param
+    // when we call getIOUReportActionDisplayMessage from ReportActionItemMessage
+    if (shouldLog) {
+        Log.alert('Transaction Param Used when getIOUReportActionDisplayMessage was called from ReportActionItemMessage', reportAction);
```

Comment:
Yeah need confirmation on the includeStackTrace from @tgolen

Reply:
If we find out this is being logged, I think the stack trace would really help us, wouldn't it? So, I think I'm in favor of keeping the stack trace. I'm not sure what value there is in logging the `reportAction`, I don't think it would tell us anything and it could contain sensitive information (like user comments).

Reply:
My intention was we could get some info about what type of report action made the code to be executed; obviously, then we can use that specific type to test. But if we can get all the important info in the stack trace, I will be good to remove it.

Reply:
I think if the report action type is what is the most important, then let's log the type specifically (or any other info that would be immediately helpful).